src/medicine/medicine.py

```diff
@@ -0,0 +1,70 @@
+class Medicine:
+    """Contains arguments necessary to calculate time after which the medicine will run out."""
+
+    def __init__(self, name: str, dose: int | float, daily_dose: int | float, current_amount: int):
+        """Create a medicine instance.
+
+        :param str name: The name of the medicine
+        :param int|float dose: The dose (in mg) in a single pill
+        :param int|float daily_dose: The dose (in mg) to be taken daily
+        :param int current_amount: the current amount of pills available
+        """
```

Comment:
sprawdzaj dane wejściowe w jednym miejscu, potem zakładaj że jesteś kompetentny na tyle żeby typów ogarniać :D 
Inne podejście to robić to w funkcji która czyta z pliku, ale w tym przypadku chyba ładniej tutaj

Reply:
Ok, nie znałem takiej konstrukcji, dzięki :)